megamek/src/megamek/client/ui/swing/util/MegaMekController.java

```diff
@@ -105,27 +92,18 @@ public boolean dispatchKeyEvent(KeyEvent evt) {
 
         int keyCode = evt.getKeyCode();
         int modifiers = evt.getModifiersEx();
-        // Get a collection of key/cmd binds that match the keycode/modifiers
-        ArrayList<KeyCommandBind> kcbs = KeyCommandBind.getBindByKey(keyCode,
-                modifiers);
 
         // If there's no action associated with this key bind, or the
         // current action is invalid, do not consume this event.
         boolean consumed = false;
 
-        for (KeyCommandBind kcb : kcbs) {
-            // Do we have a binding for this key?
-            if (!keyCmdSet.contains(kcb)) {
+        for (var kcb : KeyCommandBind.getBindByKey(keyCode, modifiers)) {
```

Comment:
It doesn't affect functionality, but, in general, I prefer to leave this sort of thing as it was previously (declare and assign variable then use it in a loop), the reason being that I can more easily, while debugging, examine the value that the function call produces.